### DISCUSSION TITLE
Change HTML Generated

### DISCUSSION THREAD

**Discussion by Fluffy9** [posativ/isso#780] [Q&A]
I want to move where the vote buttons are. The API documentation is incomplete and I can't find where the html comes from in the source.

**Comment by khiemnd179** [comment]
You can do it by create your own frontend. Or change jade file at js/app/text/postbox.jade

**Comment by Fluffy9** [comment]
Ok is there an example I can work off of for making my own. Rebuilding isso is going to be too much

**Comment by khiemnd179** [comment]
I'm also working on it now. By create new front-end use reactjs.

**Comment by Fluffy9** [comment]
Mind posting a link at some point? It would help to see an example

**Comment by ix5** [comment]
@Fluffy9 is this issue still current? Since it's more of a call for help to extend/modify isso, would you mind closing it?
